ovos_config/recommends/offline_stt/da-dk.conf

```diff
@@ -0,0 +1,9 @@
+{
+  "stt": {
+    "module": "ovos-stt-plugin-fasterwhisper",
+    "fallback_module": "",
+    "ovos-stt-plugin-fasterwhisper": {
+        "model": "small"
```

Comment:
will want to use whisper-turbo here, but that should be updated across all languages and I'll do it in a follow up PR